client/src/components/Notices/Bulletin.js

```diff
@@ -30,14 +31,13 @@ const Bulletin = ({items, number, year, date}: Props) => (
     <Table responsive className="bulletin-table">
       <thead>
         <tr>
-          <th />
           <th>Názov obstarávania</th>
           <th>Objednávateľ</th>
-          <th>Kto by sa mal prihlásiť</th>
+          <th>Kto by sa mal prihlásiť/výherca</th>
```

Comment:
Just an opinion: don't we want to split this into two columns and show winner for completed and "who should join" for pending Notices? I think having these two things in a common column makes it totally unreadable. What do you think?